jekyll/_cci2/rerun-failed-tests-only.adoc

```diff
@@ -241,6 +241,23 @@ Example:
 
 The snippet above will write the list of test file names to `files.txt`.  On a non-rerun, this list will be all of the test file names.  On a "re-run", the list will be a subset of file names (the test file names that had at least 1 test failure in the previous run).  You can pass the list of test file names from `files.txt` into, for example, your custom `makefile`.
 
+[#configure-a-job-running-playwright-tests]
+=== Configure a job running Django tests
+
+Django takes as input test filenames with a format that uses dots ("."), however, it outputs JUnit XML in a format that uses slashes "/".  To account for this: get the list of test filenames first, massage the filenames to be separated by dots "." instead of slashes "/", and pass the filenames into the test command.
```

Comment:
```suggestion
Django takes as input test filenames with a format that uses dots ("."), however, it outputs JUnit XML in a format that uses slashes "/".  To account for this, get the list of test filenames first, change the filenames to be separated by dots "." instead of slashes "/", and pass the filenames into the test command.
```